app/hotels/src/singleHotel/paymentScreen/__tests__/BookingPaymentScreen.test.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import { BookingPaymentScreen } from '../BookingPaymentScreen';
+
+const ONE_DAY = 8.64e7; // ms
+
+it('creates correct URL', () => {
+  expect(
+    new BookingPaymentScreen({
+      hotelId: '1',
+      checkin: new Date(ONE_DAY),
+      checkout: new Date(3 * ONE_DAY), // 2 days later
+      roomConfig: [
+        {
+          roomId: '7709411_91461863_1_1_0',
```

Comment:
Is this a real ID? Or looks like a real ID? Maybe we should change that.

Reply:
Not really sure, but it has been in the code base since January